WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -503,6 +503,12 @@
     context 'when signed in as results team member' do
       sign_in { FactoryGirl.create(:results_team) }
 
+      # Posts should always be in English, therefore we want to check using an English text,
+      # even if the user posting has a different locale
+      before :each do
```

Comment:
Actually no (but I didn't check why) !
I figured this out when I changed the locale only in the first test: at this point the second one (executed after) would pass. When I switched it to a "before each" it failed (leading to [this](https://github.com/thewca/worldcubeassociation.org/pull/1133/files#diff-5fa52fb0ac39e3bd45fa449ab21c9404L122).
So I concluded that there was a new session at every test.